internal/commands/push/inputs_test.go

```diff
@@ -19,7 +19,7 @@ func TestPushInputsResolve(t *testing.T) {
 		defer teardown()
 
 		var i inputs
-		assert.Equal(t, errProjectNotFound{}, i.Resolve(profile, nil))
+		assert.Equal(t, errProjectInvalid{path: profile.WorkingDirectory, pathExists: true}, i.Resolve(profile, nil))
```

Comment:
the shorthand (read: no need for `fieldName: ` declarations) syntax works well in tests, b/c presumably even if we inadvertently switch the ordering of the fields, the test will still fail